mysql-compatibility.md

```diff
@@ -95,13 +95,10 @@ See also: [TiDB SQL Grammar](https://pingcap.github.io/sqlgram/#functioncallkeyw
 In TiDB, all supported DDL changes are performed online. Compared with DDL operations in MySQL, the DDL operations in TiDB have the following major restrictions:
 
 * Multiple operations cannot be completed in a single `ALTER TABLE` statement. For example, it is not possible to add multiple columns or indexes in a single statement. Otherwise, the `Unsupported multi schema change` error might be output.
-* Different types of indexes (`HASH|BTREE|RTREE|FULLTEXT`) are not supported, and will be parsed and ignored when specified.
-* Adding/Dropping the primary key of the `CLUSTERED` type is unsupported. For more details about the primary key of the `CLUSTERED` type, refer to [clustered index](/clustered-indexes.md).
-* Changing the field type to its superset is unsupported. For example, TiDB does not support changing the field type from `INTEGER` to `VARCHAR`, or from `TIMESTAMP` to `DATETIME`. Otherwise, the error information `Unsupported modify column: type %d not match origin %d` might be output.
-* Change/Modify data type does not currently support "lossy changes", such as changing from BIGINT to INT.
-* Change/Modify decimal columns does not support changing the precision.
-* Change/Modify integer columns does not permit changing the `UNSIGNED` attribute.
+* Some type changings through `ALTER TABLE` statement is not supported. For example, TiDB does not support changing the field type from `DECIMAL` to `DATE`. Otherwise, the error information `Unsupported modify column: type %d not match origin %d` might be output. See [`ALTER TABLE`](/sql-statements/sql-statement-alter-table.md) for further details.
```

Comment:
```suggestion
* Some type changings through `ALTER TABLE` statement are not supported. For example, TiDB does not support changing the field type from `DECIMAL` to `DATE`. When a type change is not supported, the error  `Unsupported modify column: type %d not match origin %d` will be returned. See [`ALTER TABLE`](/sql-statements/sql-statement-alter-table.md) for further details.
```

Reply:
Thanks, for your suggestion, it has been addressed in @AilinKid 's https://github.com/pingcap/docs-cn/pull/6413.